dev-haskell/beam-core/beam-core-0.9.0.0.ebuild

```diff
@@ -15,10 +15,10 @@ SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
 LICENSE="MIT"
 SLOT="0/${PV}"
 KEYWORDS="~amd64 ~x86"
-IUSE="werror"
+IUSE=""
```

Comment:
You can delete this entirely when it's empty.

Reply:
Ok, good to know

Reply:
You _can_ delete when empty, but hackport with generate an empty `IUSE=""` anyway on the next bump. I'd suggest don't worry about deleting an empty `IUSE=""`.

Reply:
Fair enough.

Although also, maybe hackport shouldn't put an empty one in there...